src/app/main/component/ubs/components/ubs-confirm-page/ubs-confirm-page.component.ts

```diff
@@ -1,10 +1,19 @@
 import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 
 @Component({
   selector: 'app-ubs-confirm-page',
   templateUrl: './ubs-confirm-page.component.html',
   styleUrls: ['./ubs-confirm-page.component.scss']
 })
 export class UbsConfirmPageComponent {
+  orderId: string;
+  responseStatus: string;
 
+  constructor(private route: ActivatedRoute) {
+    this.route.queryParams.subscribe(params => {
```

Comment:
let's have unsubscribe here as well 

https://stackoverflow.com/questions/41138081/do-i-have-to-unsubscribe-from-activatedroute-e-g-params-observables